ast/ast_test.go

```diff
@@ -11,6 +11,11 @@ import (
 	"github.com/cedar-policy/cedar-go/types"
 )
 
+type CustomString string
```

Comment:
Can these types be put inside the `TestASTByTable` function?  Other than that, LGTM.

Reply:
Thanks. Moved